docs/METRICS.md

```diff
@@ -183,4 +193,35 @@ If this value is beyond the alarm intervals an alarm should have been sent.
 The value represents the effective remaining repair time for the table to be fully repaired in milliseconds.
 This is the time ecChronos will have to wait for Cassandra to perform repair,
 this is an estimation based on the last repair of the table.
-The value should be `0` if there is no repair ongoing for this table.
\ No newline at end of file
+The value should be `0` if there is no repair ongoing for this table.
+
+| t          | count | mean_rate | m1_rate  | m5_rate  | m15_rate | rate_unit     |
+|------------|-------|-----------|----------|----------|----------|---------------|
+| 1660042648 | 0     | 0.000000  | 0.000000 | 0.000000 | 0.000000 | events/second |
+
+\<keyspace\>.\<table\>-\<table-id\>-FailedRepairTasks
+
+
+| t          | count | mean_rate | m1_rate  | m5_rate   | m15_rate  | rate_unit     |
+|------------|-------|-----------|----------|-----------|-----------|---------------|
+| 1660213991 | 102   | 0.044732  | 3.000000 | 14.000000 | 42.000000 | events/second |
+
+\<keyspace\>.\<table\>-\<table-id\>-SucceededRepairTasks
+
+The count represents the total amount of failed/succeeded repair tasks for the table.
+The mean rate is the rate at which events have occurred since the beginning.
+The `m1_rate`, `m5_rate` and `m_15_rate` are the rates at which events have occurred for the past 1 minute,
+5 minutes and 15 minutes.
+For example an `m1_rate` of `15` would mean that 15 events have occurred in the past minute.
+
+For `\<keyspace\>.\<table\>-\<table-id\>-FailedRepairTasks`, `m1_rate`, `m5_rate` and `m15_rate` should always be `0`
+in healthy clusters (assuming repairs are actually running).
```

Comment:
"(assuming repairs are actually running)"

Should be 0 even if repairs are not running

Something like:
" should be close to `0` in healthy clusters. A positive rate might indicate a problem and warrant a look into the logs."